geonode_mapstore_client/client/js/plugins/Share.jsx

```diff
@@ -130,7 +131,8 @@ function Share({
                 </div>
                 <div className="gn-share-panel-body">
                     <SharePageLink url={pageUrl} label={<Message msgId="gnviewer.thisPage" />} />
-                    <SharePageLink url={embedUrl} label={<Message msgId={`gnhome.${resourceType}`} />} />
+                    <SharePageLink url={embedUrl} label={<Message msgId={`gnviewer.embed${resourceType}`} />} />
+                    {resourceType === 'document' && <SharePageLink url={downloadUrl} label={<Message msgId={`gnviewer.directLink`} />} />}
```

Comment:
Probably here is better to check also if the downloadUrl isa available
```suggestion
                    {(resourceType === 'document' && !!downloadUrl) && <SharePageLink url={downloadUrl} label={<Message msgId={`gnviewer.directLink`} />} />}
```